core/src/com/unciv/ui/cityscreen/CityScreen.kt

```diff
@@ -160,20 +188,37 @@ class CityScreen(
     }
 
     private fun updateTileGroups() {
-        val nextTile = city.expansion.chooseNewTileToOwn()
+        fun isExistingImprovementValuable(tileInfo: TileInfo, improvement: TileImprovement): Boolean {
```

Comment:
Change "improvement" to "improvementPendingConsttruction" or something, the function and parameter names give the wrong idea

Reply:
Correct (I allow myself some sloppiness in such a case (like naming a variable a, b, i, j, k, x,...) because it's a local sub-function and all uses (one) are just twelve lines away).